cockatrice/src/sequenceEdit/secuenceedit.cpp

```diff
@@ -0,0 +1,150 @@
+#include "secuenceedit.h"
+#include "../settingscache.h"
+#include <QLineEdit>
+#include <QPushButton>
+#include <QHBoxLayout>
+#include <QEvent>
+#include <QKeyEvent>
+#include <QToolTip>
+
+SecuenceEdit::SecuenceEdit(QString name, QWidget *parent) : QWidget(parent)
+{
+    this->shorcutName = name;
```

Comment:
is "this->" needed? i can see the ctor in .h is defined as ``` SecuenceEdit(QString shorcutName, QWidget *parent = 0);```, but here the first parameter has been renamed as "name", so you should not have problems with the scope of the variable.
In other places in the code, i saw this pattern when dealing with such a case:
```
void Foo::bar(QString _variable) {
    variable = _variable;
}
```
or, in this specific case of a ctor: 
```
void Foo::Foo(QString _variable)
: variable(_variable)
 {
...
}
```